apps/web/src/pages/integrations/components/ConnectIntegrationForm.tsx

```diff
@@ -119,6 +119,7 @@ export function ConnectIntegrationForm({
                   label={credential.displayName}
                   required
                   placeholder={credential.displayName}
+                  description={credential.credentialDescription ? credential.credentialDescription : ''}
```

Comment:
```suggestion
                  description={credential.description ?? ''}
```
Nullish coalescing operator can be used here :)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Cool